systemd/ubuntu-advantage.service

```diff
@@ -9,7 +9,7 @@
 [Unit]
 Description=Ubuntu Advantage GCP Auto Attach Daemon
 Documentation=man:ubuntu-advantage https://ubuntu.com/advantage
-After=network.target network-online.target systemd-networkd.service ua-auto-attach.service cloud-config.service
+After=network.target network-online.target systemd-networkd.service ua-auto-attach.service cloud-config.service ubuntu-advantage-cloud-id-shim.service
```

Comment:
+1 I was thinking there'd be a race if we didn't add After=ubuntu-advantage-cloud-id-shim.service to ubuntu-advantage.service.